typespec-extension/src/code-model-builder.ts

```diff
@@ -871,6 +871,10 @@ export class CodeModelBuilder {
   private processLroMetadata(op: CodeModelOperation, httpOperation: HttpOperation): LongRunningMetadata {
     const operation = httpOperation.operation;
 
+    // ARM uses the response of activation request as poll/final result.
+    // Therefore these schema will not be public.
+    const trackConvenienceApi: boolean = Boolean(op.convenienceApi) && !this.isArm();
```

Comment:
So ARM will not use `getLroMetadata` to calculate poll/final result? Will `getLroMetadata` enhance later to support ARM as well?

Reply:
ARM still uses `getLroMetadata`.

I may need to double check whether in all ARM cases (CRUD+action), the response of activation request is always same as the poll/result response of that `getLroMetadata`. If it is true, we only need to modify the DPG path.

Reply:
ok, I see.

Reply:
Appears I can unify the ARM/DPG (to DPG code flow), without affect existing ARM cases.

Hope this can stay correct while we add new ARM cadl-ranch cases.